src/components/ChangePasswordModal.tsx

```diff
@@ -51,52 +51,54 @@ export default function ChangePasswordModal({
   return (
     <PopUp
       closeButton
-      title={t('account.resetPassword.modal.title')}
+      title={t('changePassword.title')}
       {...modalProps}
       onHide={handleClose}
     >
-      <div>
-        <p>{t('account.resetPassword.modal.intro')}</p>
-      </div>
       <form onSubmit={handleSubmit(onSubmit)} className="vertical-rhythm">
+        <p>{t('changePassword.intro')}</p>
         <Input
-          label={t('account.resetPassword.modal.currentPassword')}
+          label={t('changePassword.currentPassword')}
           type="password"
           {...validationProps('password')}
           {...register('password', {
-            required: t('account.resetPassword.modal.currentPasswordRequired'),
-            minLength: {
```

Comment:
is the mineLength field not required here because this is the field for the current password?

Reply:
Yeah it felt like a weird validation here. Like if they were trying to guess the password, it just sort of serves as an extra hint. No real security impact, but still.